webview/src/plots/components/App.test.tsx

```diff
@@ -2798,172 +2777,6 @@ describe('App', () => {
       expect(clickEvent.stopPropagation).toHaveBeenCalledTimes(1)
     })
 
-    it('should have a tooltip on the plot if the title is cut', () => {
```

Comment:
[F] See [this](https://github.com/iterative/vscode-dvc/pull/4734/files#r1405504743)